src/runtime/pkg/katautils/config.go

```diff
@@ -58,6 +58,9 @@ const (
 	// the maximum amount of PCI bridges that can be cold plugged in a VM
 	maxPCIBridges uint32 = 5
 
+	maxPCIeRootPorts   uint32 = 16
```

Comment:
Can you add some comments on why we set the max ports to 16?

Reply:
@lifupan https://github.com/kata-containers/kata-containers/blob/main/docs/design/kata-vra.md#hypervisor-resource-limits